naga/src/front/wgsl/error.rs

```diff
@@ -791,6 +790,11 @@ impl<'a> Error<'a> {
                 )],
                 notes: vec![],
             },
+            Error::UnknownExtension(span, word) => ParseError {
+                message: format!("Unknown extension: {}. See available extensions at: https://www.w3.org/TR/WGSL/#enable-extension", word),
```

Comment:
Would be nice to list supported extensions here but don't want to bring in `strum`.

Reply:
Sounds like a good thing to consider in follow-up, perhaps?

Reply:
@ErichDonGubler Sounds good to me!